resolwe/flow/managers/listener.py

```diff
@@ -211,6 +211,106 @@ def hydrate_spawned_files(self, exported_files_mapper, filename, data_id):
 
         return {"file_temp": export_fn, "file": filename}
 
+    def handle_annotate(self, obj):
```

Comment:
It would be nice to abstract the logic of `handle_*` functions and get rid of boiler parts all over the code. However, this is not required in this pr.